charts/k8s-watcher/charts/network-mapper/values.yaml

```diff
@@ -1,8 +1,8 @@
 # Local configuration for network mapper chart
 mapper:
-  repository: otterize
+  repository: public.ecr.aws/g8r5w0b5
   image: network-mapper
-  tag: latest
+  tag: v0.1.17
```

Comment:
Is the version static?

Reply:
i belive that we should definitly pin it until we test a newer version... having latest is a really bad practice especially wen we'er not releasing it

Reply:
Cool, i thought we are building the image